test/functionalTest/cases/1048_subscription_cache/three_subscriptions.test

```diff
@@ -268,7 +282,7 @@ Date: REGEX(.*)
 06. Statistics
 ==============
 HTTP/1.1 200 OK
-Content-Length: 488
+Content-Length: 51REGEX((3|5))
```

Comment:
Mmmm... this is dirty. I have to re-thought my previous comment on "REGEX((1|2)) is the best solution" :)